src/ts/components/account/ConfirmPhrase.tsx

```diff
@@ -68,9 +67,9 @@ class ConfirmPhrase extends React.Component<IConfirmPhraseProps, IConfirmPhraseS
             console.log('Account created! ', keyringPair)
             this.props.setCreated(true)
             this.setState({ keyringPair: keyringPair })
-          })
+          }).catch(err => { this.props.setError(err) })
```

Comment:
In the catch statement, is `err` an object or a string?

Reply:
it is a message string.